datadog/data_source_datadog_synthetics_test_.go

```diff
@@ -52,7 +54,8 @@ func dataSourceDatadogSyntheticsTestRead(ctx context.Context, d *schema.Resource
 		return utils.TranslateClientErrorDiag(err, httpresp, "error getting synthetic tests")
 	}
 
-	searchedId := d.Get("test_id").(string)
+	urlRegex := regexp.MustCompile(`https:\/\/(.*)\.datadoghq\.com\/synthetics\/details\/`)
```

Comment:
```suggestion
	urlRegex := regexp.MustCompile(`https:\/\/(.*)\.(datadoghq|ddog-gov)\.(com|eu)\/synthetics\/details\/`)
```
some of ddog's sites aren't covered with `.com` or `datadoghq`

Reply:
that's a good shout, thanks.